torch/csrc/cudnn/Conv.cpp

```diff
@@ -225,6 +225,11 @@ void cudnn_convolution_forward(
   size_t workspaceSize;
   CHECK(cudnnGetConvolutionForwardWorkspaceSize(handle, idesc.desc, wdesc.desc,
       cdesc.desc, odesc.desc, fwdAlg, &workspaceSize));
+  if (workspaceSize > 1024*1024*1024) {
```

Comment:
Hi ngimel, can I use the code like this:
cudaDeviceProp deviceProp;
cudaGetDeviceProperties(&deviceProp, 0);
if (workspaceSize > deviceProp.totalGlobalMem) {